### ISSUE TITLE
FastMail send methods use sync code

### ISSUE THREAD

**Issue by uriyyo** [marlin-dev/fastapi-mail#5]
FastMail `send` methods use sync code from `smtplib` inside `async` functions, so it blocks event loop.
Methods `__configure_connection`, `__send_bulk` and `__send` use sync code.

For instance, this part is from `__configure_connection` method:
```python
if self._tls:
    session.starttls()

session.ehlo()
session.login(self._email, self._password)
self.session = session
```

**Comment by marlin-dev** [comment]
we are working all  issues right now, will release new version soon.

**Comment by uriyyo** [comment]
It's great to hear that🙂
Should I close this issue?

**Comment by marlin-dev** [comment]
Let it to be open, as soon we merge PR, we have to mention it.

**Comment by marlin-dev** [comment]
This now should be fixed #10 , if you face with any issues feel free to open issues. Thanks. tslightly_smiling_face
